src/nextjournal/clerk/analyzer.clj

```diff
@@ -76,8 +76,12 @@
 (defn- analyze-form
   ([form] (analyze-form {} form))
   ([bindings form]
-   (binding [config/*in-clerk* true]
-     (ana-jvm/analyze form (ana-jvm/empty-env) {:bindings bindings}))))
+   (try
+     (binding [config/*in-clerk* true]
+       (ana-jvm/analyze form (ana-jvm/empty-env) {:bindings bindings}))
+     (catch Throwable e
+       (throw (ex-info (str "cannot analyze form " form ": " (ex-message e))
+                       {:form form :bindings bindings} e))))))
```

Comment:
Can you undo this change? I'd like to have this separately and I'm unsure this is the right place for the catch, want to surface more context (the file etc).